### PR TITLE
WP-42 and WP-44 implemented search bar, removed auth0

### PR DESCRIPTION
**Pull request by G-Andrey**
ex1
ex2

- Implemented search bar

- removed auth0 as best as I could

### CODE REVIEW

File: WatchParty/client/src/components/Header/Header.js [hunk 1]
```diff
@@ -1,43 +1,96 @@
...
+                title=<FontAwesomeIcon icon='user-circle' size='2x' />
+                id="collapsible-nav-dropdown"
+                alignRight>
+                <NavDropdown.Item>Log In</NavDropdown.Item>
+              </NavDropdown>
+            }
+          </Nav>
+        </Navbar.Collapse>
+      </Navbar>
+      <Switch>
```
Comment (ian-s-mcb) [inline WatchParty/client/src/components/Header/Header.js:86]: I'm pretty sure that the `switch` component is supposed to instantiated in the root component, not every child component.

File: WatchParty/client/src/components/Header/Header.js [hunk 2]
```diff
@@ -1,43 +1,96 @@
 import React from 'react';
+import { Route, Switch, withRouter} from "react-router-dom"
 import { Nav, Navbar, NavDropdown } from 'react-bootstrap'
 import { Form, FormControl, Button } from 'react-bootstrap'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 
+import SearchResults from "./SearchResults"
```
Comment (ian-s-mcb) [inline WatchParty/client/src/components/Header/Header.js:7]: Shouldn't the `SearchResults` component be instantiated in the `App` component? I know sure why it's being lumped together with `Header`

File: WatchParty/client/src/components/Header/Header.js [hunk 3]
```diff
@@ -1,43 +1,96 @@
...
+
+    } else {
+      alert("Please enter a movie name!")
+    }
+
+  }
+
+  handleFormSubmit(event){
+    event.preventDefault();
+  }
```
Comment (ian-s-mcb) [inline WatchParty/client/src/components/Header/Header.js:52]: Couldn't this be implemented as a one-liner, in the render() method?

**Comment by ian-s-mcb** [comment]
This is a good start but I think some refactoring is need before it can be merged.

File: WatchParty/client/src/components/Header/Header.js (hunk 3 above)
Comment (G-Andrey) [reply WatchParty/client/src/components/Header/Header.js]: fixed

**Comment by G-Andrey** [comment]
Ive looked into routing a bunch more and looking back at my work it does seem all over the place as I was mostly concerned with getting search functionality working. Ill work on refactoring it.

**Comment by ian-s-mcb** [comment]
@G-Andrey Alright. Let us know you're ready for us to look at your work.

**Comment by shahan27** [comment]
Running with this issue @G-Andrey when I try to run the server. Is there something I'm missing? 
Screen Shot 2020-04-09 at 4 39 24 AM

**Comment by ian-s-mcb** [comment]
@G-Andrey Looks good to me.

@shahan27 That Node package was added a few PR's ago. If you run `npm install` in the client folder, you should be fine.

**Comment by PredatorFeesh** [comment]
Are we ready to merge this into sandbox?

**Pull request merged**